src/test/java/Homework_25.java

```diff
@@ -0,0 +1,25 @@
+import PageObject.LoginPage;
+
+import org.testng.annotations.Test;
+
+import static java.sql.DriverManager.getDriver;
+
+public class Homework_25 extends BaseTest {
+    @Test
+    public void loginWithValidCredentials() throws Throwable {
```

Comment:
Please, delete `Throwable`

Reply:
![throwable](https://github.com/Saty-lab/A53-QA-Automation-Framework/assets/149276552/bab6eeaa-7143-45b0-b4bd-14c7346be17e)
Throwable is needed for this method.

Reply:
![Throwable](https://github.com/Saty-lab/A53-QA-Automation-Framework/assets/149276552/53ff8236-92b0-4c95-a60c-8233b1e6ee07)
Throwable is needed for this method @Berichicko